nlp_primitives/tests/test_mean_characters_per_word.py

```diff
@@ -41,7 +41,31 @@ def test_nans(self):
                        '',
                        'third line'])
         primitive_func = self.primitive().get_function()
-        answers = pd.Series([np.nan, np.nan, 4.5])
+        answers = pd.Series([np.nan, 0, 4.5])
+        pd.testing.assert_series_equal(primitive_func(x), answers, check_names=False)
+
+    def test_all_nans(self):
```

Comment:
Could we test this with `pd.NA` instead of `np.nan` as a `NaturalLanguage` column with use the pandas `string` dtype which should result in `pd.NA` values being a passed to the primitive function.

Could also maybe parameterize a new test replacing `test_all_nans` and `test_none` to check with `None`, `pd.NA` and `np.nan` as they should all produce the same result.